src/gltf-component-mappings.js

```diff
@@ -43,7 +46,13 @@ AFRAME.GLTFModelPlus.registerComponent(
     };
   })()
 );
-AFRAME.GLTFModelPlus.registerComponent("visible", "visible");
+AFRAME.GLTFModelPlus.registerComponent("visible", "visible", (el, componentName, componentData) => {
```

Comment:
How come the `visible` component needs this custom `inflator`?

Reply:
Same as above, Spoke cannot export components with literal prop values. Prop values are always objects so we need to use a custom inflator to translate the value to a-frame but also maintain backwards compat for our existing environments (which we will eventually convert to the new format).